src/mobile/Studio/components/WorkList.tsx

```diff
@@ -0,0 +1,127 @@
+import { css } from '@emotion/react';
+import { colors, fonts } from '../../../styles/theme';
+
+import { DESIGNER_TOTAL_WORKS } from '../../constants/WORKS';
+
+function WorkList() {
+  const workList = DESIGNER_TOTAL_WORKS.data.works;
+
+  return (
+    <ul css={listContainer}>
+      {workList.map((work) => {
+        const { workId, workTitle, images, designers } = work;
+        const imgSrc =
+          images.length === 2 ? images[1].imgPath : images[0].imgPath;
```

Comment:
아래처럼 코드 수정하면 더 깔끔하게 구현할 수 있어요! (물론 이 부분은 크게 차이가 있는 부분은 아니니 반영하지 않아도 됩니다!)

```suggestion
        const { imgPath } =
          images.length === 2 ? images[1] : images[0];
```